packages/atomic/src/components/facets-v1/facet-date-input/facet-date-input.tsx

```diff
@@ -0,0 +1,128 @@
+import {Component, h, State, Prop, Event, EventEmitter} from '@stencil/core';
+import {DateFilter, DateFilterState, buildDateRange} from '@coveo/headless';
+import {Bindings} from '../../../utils/initialization-utils';
+import dayjs from 'dayjs';
+
+/**
+ * Internal component made to be integrated in a TimeframeFacet.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-facet-date-input',
+  shadow: false,
```

Comment:
Why not use a shadowDOM here?

Reply:
Its inside a facet which already has shadow dom